web-wallet/src/lib/components/Send/Send.svelte

```diff
@@ -160,6 +160,21 @@
           type="multiline"
           bind:value={address}
         />
+        {#if addressValidationResult.type === "account"}
+          <div class="warning-banner" transition:fade>
```

Comment:
This will eventually be moved to a reusable `InfoPannel` component.

Issue created here https://github.com/dusk-network/rusk/issues/2696